packages/opa-react/tests/use-authz.test.tsx

```diff
@@ -1,30 +1,61 @@
 import React from "react";
+import { QueryClient, QueryFunctionContext } from "@tanstack/react-query";
 import { renderHook, waitFor } from "@testing-library/react";
 import { describe, it, expect, vi } from "vitest";
-import { OPAClient, type Result } from "@styra/opa";
-import AuthzProvider, { AuthzProviderProps } from "../src/authz-provider";
+import { OPAClient, type Input, type Result } from "@styra/opa";
+import { AuthzContext, AuthzProviderProps } from "../src/authz-provider";
 import useAuthz from "../src/use-authz";
 
 describe("useAuthz Hook", () => {
   const opa: OPAClient = Object.assign(new OPAClient(""), {
     evaluate: vi.fn(),
     evaluateDefault: vi.fn(),
   });
+  // TODO(sr): this duplication is not optimal; but since the inner function is mocked, we should just mock here perhaps?
```

Comment:
The only difference is `retries: false` down below. I might have to look up "don't import this from the outside" conventions, like a filename of`internal/default_query_client.ts` or something like that 🤔

Reply:
✔️